config/config-defaults-triggers.yaml

```diff
@@ -42,3 +42,4 @@ data:
     default-service-account: "default"
     default-run-as-user: "65532"
```

Comment:
i think we should default set them empty, and if user wants they can set as their desired value

Reply:
Those values were set from starting and now if we remove we might break behavior
IMO its okay to set some default values when user won't set anything

We are setting default values here https://github.com/tektoncd/triggers/pull/1747/files#diff-572521168fb33345a93f07ab72e633d00cccb422144d8a05399a6fa98cb12025R33-R35

In the config-defaults-triggers.yaml is sample example to show how user can set as its under `_example: |`